kebechet/managers/update.py

```diff
@@ -353,7 +367,7 @@ def _create_initial_lock(self, labels: list, pipenv_used: bool) -> bool:
         branch_name = "kebechet-initial-lock"
```

Comment:
@fridex While testing: if requirements.txt and requirements-dev.txt were not present, only both requirements.in and requirements-dev.in files where present. Then kebechet creates a PR with branch `kebechet-initial-lock` for requirements.txt but fails to create another PR as the first is still existing.
It is highly unlikely that a repo doesn't have .txt files and has both .in files but still, it can occur, a certain problem may occur:
If we are planing to first process requirements.in then requirements-dev.in then the problem is that we wont be able to get the requirements-dev.txt once PR is generated for requirements.txt

Reply:
Let's do initial lock only for `requirements.in` first, once that PR is merged, kebechet can proceed to `requirements-dev.txt` to bypass this branch collision.